recipes/goosempl/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "goosempl" %}
+{% set version = "0.2.25" %}
+{% set sha256 = "ca21f592707bee038330a663021ae5c4c22efd68c6ee20fdcce0eb31c8422cb3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/tdegeus/GooseMPL/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+
+requirements:
+  host:
+    - python >=3
+    - pip
+    - setuptools
+  run:
+    - python >=3
+    - setuptools
+    - numpy
+    - matplotlib
```

Comment:
Should this be `matplotlib-base`?

Reply:
@scopatz Looking at https://github.com/conda-forge/matplotlib-feedstock/blob/master/recipe/meta.yaml I guess you are right. Out of curiosity: Why is the package called `matplotlib-base` and not `matplotlib`? Any why isn't the feedstock called `matplotlib-base`?

Reply:
It probably used to only contain `matplotlib` and the split that created `matplotlib-base` was probably added afterwards.

It's then a quirk of the way multiple `outputs` interact with the README is automatically generated.